sdk/storage/azure-storage-blob-changefeed/azure/storage/blob/changefeed/_change_feed_client.py

```diff
@@ -88,6 +88,7 @@ def list_changes(self, **kwargs):
 
         :keyword datetime start_time:
             Filters the results to return only events which happened after this time.
+            If start_time and continuation_token are both set, start_time will be ignored.
```

Comment:
Shouldn't we throw some validation error if both are provided? Silent fallback might be harder to debug.

Reply:
I was thinking if users want to use live mode and list events every one minute starting from a specific time, the current code would make it complicated to implement. When I added a top level while loop in samples/change_feed_samples.py, I feel remove the validation error could be more user friendly.

Reply:
User should be able to initialize CF with start date and then use continuation in subsequent iterations. but that should be two separate calls. Accepting two arguments that mean same (both start time and continuation point to starting point) and ignoring one of them isn't great. We should avoid this kind of implicit rules. 
Take a look at .NET sample for reference https://github.com/Azure/azure-sdk-for-net/blob/5dbcc86da8b60ddec313455ddb6cb9d72b5175a3/sdk/storage/Azure.Storage.Blobs.ChangeFeed/samples/Sample01b_HelloWorldAsync.cs#L132-L155

Reply:
Yeah I agree with Kamil here, in Java at least we have 3 methods - no start time or end time, one with start and end time (optional) and one with cursor

Reply:
I put the reason here https://github.com/Azure/azure-sdk-for-python/pull/13243#discussion_r482803529